### PR TITLE
Enhancement: Enable `binary_operator_spaces` fixer

### PR DESCRIPTION
**Pull request by localheinz**
This pull request

- [x] enables the `binary_operator_spaces` fixer
- [x] runs `make coding-standards`

Follows #559.

💁‍♂️ For reference, see https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v3.4.0/doc/rules/operator/binary_operator_spaces.rst.

### CODE REVIEW

File: cal.php [hunk 1]
```diff
@@ -363,17 +363,17 @@ function read_event($fp)
...
-        'sdesc'     => $sdesc,
-        'url'       => $url,
-        'ldesc'     => base64_decode($ldesc, false),
-        'country'   => $country,
-        'category'  => $category,
+        'sdesc' => $sdesc,
+        'url' => $url,
+        'ldesc' => base64_decode($ldesc, false),
+        'country' => $country,
+        'category' => $category,
```
Comment (cmb69) [inline cal.php:376]: I don't like such vertical alignments, but others may, so let's wait a while for possible objections.

**Review by cmb69 — APPROVED** [review]
Thank you!

File: cal.php (hunk 1 above)
Comment (localheinz) [reply cal.php:376]: @cmb69 

For the record, it is also possible to configure the spacing per operator: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v3.4.0/doc/rules/operator/binary_operator_spaces.rst.

**Comment by cmb69** [comment]
No objections, so I'm going to merge. Thank you, @localheinz!

**Comment by localheinz** [comment]
Thank you, @cmb69!

**Pull request merged**